ethereum/contracts/relayer/coreRelayer/CoreRelayerBase.sol

```diff
@@ -0,0 +1,264 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.19;
+
+import {IWormhole} from "../../interfaces/IWormhole.sol";
+import {IRelayProvider} from "../../interfaces/relayer/IRelayProvider.sol";
+import {toWormholeFormat, min, pay} from "./Utils.sol";
+import {
+  NoDeliveryInProgress,
+  ReentrantDelivery,
+  ForwardRequestFromWrongAddress,
+  RelayProviderDoesNotSupportTargetChain,
+  RelayProviderQuotedBogusAssetPrice,
+  Send,
+  DeliveryInstruction,
+  ExecutionParameters,
+  IWormholeRelayerBase
+} from "../../interfaces/relayer/IWormholeRelayer.sol";
+import {
+  ForwardInstruction,
+  DeliveryTmpState,
+  getDeliveryTmpState,
+  getRegisteredCoreRelayersState
+} from "./CoreRelayerStorage.sol";
+
+abstract contract CoreRelayerBase is IWormholeRelayerBase {
+  //TODO AMO: see https://book.wormhole.com/wormhole/3_coreLayerContracts.html#consistency-levels
```

Comment:
agreed that these should be provided by IWormhole. In lieu of that, is 15 the best choice?

Reply:
Yes, @chase-45 signed off on it